styles/styles.css

```diff
@@ -328,6 +328,7 @@ a:any-link {
   color: var(--color-blue);
   text-decoration: none;
   background-color: transparent;
+  font-size: 1rem;
```

Comment:
@JiangLong2019  does it work well with other links on the page ?

Reply:
@anagarwa Not all but most of the orange buttons work as this. Additionally, I'm thinking about making this as common styling so that we don't need to add same button css block by block. Such as, 
<img width="804" alt="image" src="https://github.com/aemsites/icicidirect/assets/57093988/14af2a44-cd27-425d-8d22-2a41406103db">